content/docker-hub/convert-account.md

```diff
@@ -28,10 +26,24 @@ Before you convert a user account to an organization, ensure that you meet the f
 
     If the user account is the sole owner of any organization or company, assign another user the owner role and then remove yourself from the organization or company.
 
--  You must have a separate Docker ID ready to assign it as the owner of the organization during conversion.
+-  You must have a separate Docker ID ready to assign as the owner of the organization during conversion.
 
     If you want to convert your user account into an organization account and you don't have any other user accounts, you need to create a new user account to assign it as the owner of the new organization. With the owner role assigned, this user account has full administrative access to configure and manage the organization. You can assign more users the owner role after the conversion.
 
+## Effects of converting an account into an organization
```

Comment:
Nit:
Can this section be made more active?
'Conversion has the ...' '...account is removed..' '...owners receive...' etc?